### PR TITLE
TUP-27849

### PR DESCRIPTION
**Pull request by hzhao-talendbj**

### CODE REVIEW

File: main/plugins/org.talend.core.runtime/src/main/java/org/talend/core/database/conn/version/EDatabaseVersion4Drivers.java [hunk 1]
```diff
@@ -107,7 +107,7 @@
 
     PLUSPSQL_PRIOR_TO_V9(new DbVersion4Drivers(EDatabaseTypeName.PLUSPSQL,
```
Comment (bhe-talendbj) [inline main/plugins/org.talend.core.runtime/src/main/java/org/talend/core/database/conn/version/EDatabaseVersion4Drivers.java:108]: 1 - According to https://jdbc.postgresql.org/download.html, 42.2.14 supports PostgreSQL 8.2 or newer and requires Java 6 or newer. Can we Remove postgresql-8.4-703.jdbc4.jar as well?

File: main/plugins/org.talend.libraries.jdbc.postgresql/build.properties [hunk 2]
```diff
@@ -1,5 +1,4 @@
 output.. = bin/
 bin.includes = META-INF/,\
                .,\
-               lib/postgresql-8.4-703.jdbc4.jar,\
-               lib/postgresql-9.4-1201.jdbc41.jar
+               lib/postgresql-8.4-703.jdbc4.jar
```
Comment (bhe-talendbj) [inline main/plugins/org.talend.libraries.jdbc.postgresql/build.properties:4]: Looks like missing postgresql-42.2.14.jar ?

File: main/plugins/org.talend.core.runtime/src/main/java/org/talend/core/database/conn/version/EDatabaseVersion4Drivers.java (hunk 1 above)
Comment (hzhao-talendbj) [reply main/plugins/org.talend.core.runtime/src/main/java/org/talend/core/database/conn/version/EDatabaseVersion4Drivers.java:108]: removed  postgresql-8.4-703.jdbc4.jar and relative codes.

File: main/plugins/org.talend.libraries.jdbc.postgresql/build.properties (hunk 2 above)
Comment (hzhao-talendbj) [reply main/plugins/org.talend.libraries.jdbc.postgresql/build.properties]: added into build.properties

**Pull request merged**